### ISSUE TITLE
MQTT function

### ISSUE THREAD

**Issue by italoc** [mongoose-os-apps/shelly-homekit#24]
with this firmware is possible to mantain MQTT function?

**Comment by andyblac** [comment]
I believe not, it is homekit only, it more secure that way.

**Comment by sja** [comment]
I would also like to still have MQTT support. Think about using switch functionality with HomeKit UI and reading measurements with MQTT in some sort of persistence or Home Automation solution.

**Comment by rojer** [comment]
makes sense. it would be possible, though we are currently very tight on space, we'd need to solve that first.

**Comment by pnuding2** [comment]
I'd love this, too. Especially useful with power measurement, where Homekit isn't very helpful

**Comment by timoschilling** [comment]
@pnuding2 we already expose the power measurement to HomeKit via the Eve specification, try the Eve app or Home+ App to view them or creating related automatisations.

**Comment by opctim** [comment]
@rojer How is it going with this? I would really like to see an mqtt feature on shelly-homekit! Would also be nice to have a click counter on the devices so we can see in mqtt how many times the switch was pressed in a given timespan (milliseconds) to switch other devices on "doubleclick"

**Comment by rojer** [comment]
no update on MQTT

**Comment by kaese2002** [comment]
still request that as well...
these are the only devices i cannot get into iobroker...
